criu/sockets.c

```diff
@@ -926,7 +926,7 @@ int collect_sockets(struct ns_id *ns)
 	req.r.i.sdiag_protocol = IPPROTO_ICMP;
 	req.r.i.idiag_ext = 0;
 	req.r.i.idiag_states = -1; /* All */
-	set_collect_bit(req.r.i.sdiag_family, req.r.i.sdiag_protocol);
+	set_collect_bit(req.r.n.sdiag_family, req.r.n.sdiag_protocol);
```

Comment:
same here... This change is applied in `zdtm: set net.ipv4.ping_group_range to 40000-50000`, but it seems unrelated.